### PR TITLE
AdButler - text ad

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: src/app/components/adbutler/adbutler.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-<table class="butler" style="width: 600px; border: 0px solid #C0C0C0; background-color: #263238" cellSpacing=0 cellPadding=3>
-  <tr><td height=80 style="background-color: #263238" #code>
+<table class="butler" height=80 [ngClass]="(theme=='dark')?'butlerdark':'butlerlight'" cellSpacing=0 cellPadding=3>
```
Comment (piotrpekala7) [inline src/app/components/adbutler/adbutler.component.html]: for future changes

**Comment by piotrpekala7** [comment]
Hard to test if calculated time is always correct

**Comment by ziajka** [comment]
@piotrpekala7 can we have a check if there is internet connection before displaying notification?

File: src/app/components/notification-box/notification-box.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="notification-box" *ngIf="isVisible">
+    <mat-progress-bar mode="determinate" [value]="progress"></mat-progress-bar>
+    <div style="display: flex; height: 80px;">
+        <div class="edgeline"></div>
+        <div class="content">
+            <app-adbutler theme="dark"></app-adbutler>
+            <mat-icon (click)="closeNotification()" class="close-button">close</mat-icon>
+            <!-- <button class="check-button" mat-button>CHECK IT OUT</button> -->
+        </div>
+        <div class="edgeline"></div>
```
Comment (ziajka) [inline src/app/components/notification-box/notification-box.component.html]: I'm wondering if in .content { border-left/right/bottom } wouldn't be sufficient.

File: src/app/components/notification-box/notification-box.component.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
+.notification-box {
+    posiTion: fixed;
```
Comment (ziajka) [inline src/app/components/notification-box/notification-box.component.scss]: Seems like typo

File: src/app/components/notification-box/notification-box.component.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    ngOnInit() {
+        this.notificationSettings = this.notifactionSettingsService.getConfiguration();
+        this.startTimer();
+    }
+
+    startTimer() {
+        this.timer = timer(this.notificationSettings.delayTime, 1000);
+
+        this.timerSubscription = this.timer.subscribe(() => {
```
Comment (ziajka) [inline src/app/components/notification-box/notification-box.component.ts]: ngDestroy may have "this.timerSubscription.unsubscribe()"

File: src/app/components/notification-box/notification-box.component.html (hunk 2 above)
Comment (piotrpekala7) [reply src/app/components/notification-box/notification-box.component.html]: Fixed

File: src/app/components/notification-box/notification-box.component.scss (hunk 3 above)
Comment (piotrpekala7) [reply src/app/components/notification-box/notification-box.component.scss]: Fixed

File: src/app/components/notification-box/notification-box.component.ts (hunk 4 above)
Comment (piotrpekala7) [reply src/app/components/notification-box/notification-box.component.ts]: Fixed

**Comment by ziajka** [comment]
@piotrpekala7 if you may please check if still works on issue with adbutler

**Comment by ziajka** [comment]
Jumping progress on FF

**Comment by ziajka** [comment]
Remove NotSettingsService

**Comment by piotrpekala7** [comment]
NotificationSettingsService removed, no unexpected behavior was observed, subscription exchanged to rxjs timer

**Comment by ziajka** [comment]
Looks nice, thanks @piotrpekala7

**Pull request closed without merging**